index.js

```diff
@@ -0,0 +1,55 @@
+const beeline = require('honeycomb-beeline');
+
+const instrumentHTTP = require('./instrumentation/http');
+
+/**
+ * @typedef {object} Options
+ * @property {string} APIKey
+ * @property {string | undefined} APIHost
+ * @property {string} TracingDataset
+ * @property {number} DesiredSampleRate
+ * @property {Record<string, unknown> | undefined} GlobalMetadata
+ */
```

Comment:
Oh, can you use jsdocs to declare types without the types actually being there?

Reply:
Yep, many of the typescript features can be used via JSDoc - https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html